etl/app/server/database.js

```diff
@@ -78,6 +78,32 @@ export async function endConnPool(pool) {
   log.info('EqPool connection pool ended');
 }
 
+async function cacheProfileStats(pool, schemaName, profileStats) {
+  const client = await getClient(pool);
+  try {
+    await client.query('BEGIN');
+    for (const profile of profileStats) {
+      await client.query(
+        'INSERT INTO logging.mv_profile_stats(profile_name, schema_name, num_rows, last_refresh_end_time, creation_date)' +
+          ' VALUES ($1, $2, $3, $4, current_timestamp)',
+        [
+          profile['name'].replace('attains_app.profile_', ''),
+          schemaName,
+          profile['num_rows'],
+          profile['last_refresh_end_time'],
+        ],
+      );
```

Comment:
```suggestion
      await client.query(
        'INSERT INTO logging.mv_profile_stats(profile_name, schema_name, num_rows, last_refresh_end_time, last_refresh_elapsed, creation_date)' +
          ' VALUES ($1, $2, $3, $4, $5, current_timestamp)',
        [
          profile['name'].replace('attains_app.profile_', ''),
          schemaName,
          profile['num_rows'],
          profile['last_refresh_end_time'],
          profile['last_refresh_elapsed'],
        ],
      );
```

Can you also add the `last_refresh_alapsed` as well? I don't think we need to display it on the screen, but I could see this information being useful for tracking the performance of refreshing the attains MVs.